### DISCUSSION TITLE
`cap_html_filtering`

### DISCUSSION THREAD

**Discussion by stephenhawk8054** [uBlockOrigin/uBlock-issues#2834] [Q&A] [answered]
Anyone knows in which version did `cap_html_filtering` release?

**Comment by uBlock-user** [comment]
Likely with Firefox 57 version which was 4 years ago.

**Comment by ghost** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/c34326cf4e191c2ee2723523d0a57fa4d9f425bd 1.16?

**Reply by stephenhawk8054** [reply]
Oh I see. It was introduced same time with HTML filtering. So the difference was between Firefox 56 & Firefox 57 back then.

In Firefox 56, `cap_html_filtering` is marked red and the filter inside is ignored: https://github.com/uBlockOrigin/uBlock-issues/assets/66517106/0b17e8c2-9e26-4fa6-9c3d-34f2c96706ac

In Firefox 57, it is marked green and the filter is available as normal: https://github.com/uBlockOrigin/uBlock-issues/assets/66517106/9461d77d-c2c5-406c-8aba-5a3d7d5bdf96

Both running with same uBO version 1.32.4

**Reply by MasterKia** [reply]
Who's using Firefox 56?

**Reply by stephenhawk8054** [reply]
Some users asked about youtube ads appear when we removed `json-prune` filters:

- https://www.reddit.com/r/uBlockOrigin/comments/16lmeri/youtube_antiadblock_and_ads_september_18_2023/k1wl8df/
- https://www.reddit.com/r/uBlockOrigin/comments/16lmeri/youtube_antiadblock_and_ads_september_18_2023/k1ts5e7/
